#use wml::debian::translation-check translation="1.2"
<define-tag description>DNS 応答詐称</define-tag>
<define-tag moreinfo>
<p>Python 言語の DNS クライアント実装 PyDNS に複数の欠陥が発見されました。
Dan Kaminsky さんにより、DNS トランザクション ID のランダムさの不足と、
UDP ソースポートランダム化が多くの DNS 実装に欠けていることから、DNS 応
答詐称とそれによるキャッシュ汚染攻撃が現実の脅威となることが示されまし
た。Scott Kitterman さんにより、python-dns がトランザクション ID とソー
スポートの何れもランダム化していないため、この予測可能であるという脆弱
性を持つことが報告されました。この両方のランダム性不足の問題のため、
python-dns を使うアプリケーションは応答詐称攻撃に対してきわめて脆弱です。</p>

<p>The Common Vulnerabilities and Exposures project はこの種の問題を
<a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-1447">CVE-2008-1447</a>
および PyDNS におけるこの特定例を
<a href="http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2008-4099">CVE-2008-4099</a>
として認識しています。</p>

<p>安定版 (stable) ディストリビューション (etch) では、これらの問題はバージ
ョン 2.3.0-5.2+etch1 で修正されています。</p>

<p>直ぐに python-dns パッケージをアップグレードすることを勧めます。</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2008/dsa-1619.data"
# $Id$
